specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/application/rfc7540/connection.management/http.push.promise/server.rpt

```diff
@@ -36,6 +36,27 @@ connected
 
 read "Hello, world"
 
+
+# First push promise
+write advise zilla:flush ${http:flushEx()
+                               .typeId(zilla:id("http"))
+                               .promiseId(promiseId1)
```

Comment:
Please default this property to `-1L` at top of script, and override to `0x3f3f000000000005L` in implementation IT instead of spec IT.